### DISCUSSION TITLE
Unit Tests - Is there a way to mock only one store?

### DISCUSSION THREAD

**Discussion by Jak3b0** [vuejs/pinia#2672] [Help and Questions] [answered]
Hi,

I currently have a store that depends on another store (Store A uses store B). I would like to only mock the 'B' store but use the real implementation of store 'A' since it's the testing subject.

When using `createTestingPinia`, store 'A' and 'B' gets mock. 

Is there a way do this?

Thank you! :)

**Comment by posva** [comment, marked as answer]
I would use a native mock (jest, vitest have docs for this). But I would also consider not mocker the stores at all and just write an integration test to avoid mocking too much and not testing anything.
